spec/system/candidate_interface/carry_over/candidate_carries_over_unsubmitted_application_with_a_course_to_new_cycle_spec.rb

```diff
@@ -135,7 +140,7 @@ def when_i_add_a_course
     click_button t('continue')
 
     expect(page).to have_content 'You’ve added Primary (2XT2) to your application'
-    expect(page).to have_content 'You can choose 2 more courses'
+    expect(page).to have_content 'You can add 2 more courses'
```

Comment:
Is this copy change independent of the AA change? Could we include in at least a separate commit?

Reply:
This is related to this change but I am happy to do in a separate commit :)

Reply:
Done! ♻️